test/parallel/test-crypto-authenticated.js

```diff
@@ -16,39 +16,111 @@ crypto.DEFAULT_ENCODING = 'buffer';
+    tag: '272b422f62eb545eaa15b5ff84092447', tampered: false },
   { algo: 'aes-128-gcm',
     key: '6970787039613669314d623455536234',
     iv: '583673497131313748307652', plain: 'Hello World!',
-    ct: '4BE13896F64DFA2C2D0F2C76', aad: '000000FF',
-    tag: 'BA2479F66275665A88CB7B15F43EB005', tampered: false },
+    ct: '4be13896f64dfa2c2d0f2c76', aad: '000000FF',
+    tag: 'ba2479f66275665a88cb7b15f43eb005', tampered: false },
   { algo: 'aes-128-gcm',
     key: '6970787039613669314d623455536234',
     iv: '583673497131313748307652', plain: 'Hello World!',
-    ct: '4BE13596F64DFA2C2D0FAC76',
-    tag: '272B422F62EB545EAA15B5FF84092447', tampered: true },
+    ct: '4be13596f64dfa2c2d0fac76',
+    tag: '272b422f62eb545eaa15b5ff84092447', tampered: true },
   { algo: 'aes-256-gcm',
     key: '337a54767a7233703637564336316a6d56353472495975313534357834546c59',
     iv: '36306950306836764a6f4561', plain: 'Hello node.js world!',
-    ct: '58E62CFE7B1D274111A82267EBB93866E72B6C2A',
-    tag: '9BB44F663BADABACAE9720881FB1EC7A', tampered: false },
+    ct: '58e62cfe7b1d274111a82267ebb93866e72b6c2a',
+    tag: '9bb44f663badabacae9720881fb1ec7a', tampered: false },
   { algo: 'aes-256-gcm',
     key: '337a54767a7233703637564336316a6d56353472495975313534357834546c59',
     iv: '36306950306836764a6f4561', plain: 'Hello node.js world!',
-    ct: '58E62CFF7B1D274011A82267EBB93866E72B6C2B',
-    tag: '9BB44F663BADABACAE9720881FB1EC7A', tampered: true },
+    ct: '58e62cff7b1d274011a82267ebb93866e72b6c2b',
+    tag: '9bb44f663badabacae9720881fb1ec7a', tampered: true },
   { algo: 'aes-192-gcm',
     key: '1ed2233fa2223ef5d7df08546049406c7305220bca40d4c9',
     iv: '0e1791e9db3bd21a9122c416', plain: 'Hello node.js world!',
     password: 'very bad password', aad: '63616c76696e',
-    ct: 'DDA53A4059AA17B88756984995F7BBA3C636CC44',
-    tag: 'D2A35E5C611E5E3D2258360241C5B045', tampered: false }
+    ct: 'dda53a4059aa17b88756984995f7bba3c636cc44',
+    tag: 'd2a35e5c611e5e3d2258360241c5b045', tampered: false },
+
+  // Following test cases are from
+  //   http://csrc.nist.gov/groups/ST/toolkit/BCM/documents/proposedmodes/gcm/gcm-revised-spec.pdf
+
+  // Test case 1
+  { algo: 'aes-128-gcm',
+    key: '00000000000000000000000000000000',
+    iv: '000000000000000000000000',
+    plain: '',
+    plainIsHex: false,
+    ct: '',
+    tag: '58e2fccefa7e3061367f1d57a4e7455a', tampered: false },
+
+  // Test case 2
+  { algo: 'aes-128-gcm',
+    key: '00000000000000000000000000000000',
+    iv: '000000000000000000000000',
+    plain: '00000000000000000000000000000000',
+    plainIsHex: true,
+    ct: '0388dace60b6a392f328c2b971b2fe78',
+    tag: 'ab6e47d42cec13bdf53a67b21257bddf', tampered: false },
+
+  // Test case 3
+  { algo: 'aes-128-gcm',
+    key: 'feffe9928665731c6d6a8f9467308308',
+    iv: 'cafebabefacedbaddecaf888',
+    plain: 'd9313225f88406e5a55909c5aff5269a86a7a9531534f7da2e4c303d8a318a721c3c0c95956809532fcf0e2449a6b525b16aedf5aa0de657ba637b391aafd255',
```

Comment:
for the sake of readability and linting, can these be split into multiple concatenated lines? e.g. 

```
'd9313225f88406e5a55909c5aff526' +
'9a86a7a9531534f7da2e4c303d8a31' +
'8a721c3c0c95956809532fcf0e2449' +
'a6b525b16aedf5aa0de657ba637b39' +
'1aafd255'
```